File: forum-client/src/Types/Post.ts
export interface Post{
    id: Number,
    title: string,
    text?: string,
    date: Date,
    user_Id: Number
    user_Username: string,
    likes: Number,
    comments: Number,
    liked: boolean
}

export interface PostInput {
    title: string,
    text?: string,
    user_Id: Number
}

File: Domain/Entities/Post.cs
using System;

namespace Domain.Entities
{
    public class Post
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Text { get; set; }
        public DateTime Date { get; set; }
        public int User_Id { get; set; }
        public string User_Username { get; set; }
        public int Likes {  get; set; }
        public int Comments {  get; set; }
        public bool Liked { get; set; }
        public Post() { }
    }
}
